docs/docs/resources/examples/word-count/pipeline.yaml

```diff
@@ -0,0 +1,28 @@
+- type: producer
+  name: wc-data-producer
+  app:
+    image: bakdata/kpops-demo-sentence-producer
+    imageTag: 1.0.0
+
+- type: streams-app
+  name: wc-word-counter
+  to:
+    topics:
+      ${output_topic_name}:
+        configs:
+          cleanup.policy: compact
+  app:
+    replicaCount: 1
+    image: bakdata/kpops-demo-word-count-app
+    imageTag: 1.0.0
```

Comment:
```suggestion
    imageTag: "1.0.0"
```

Reply:
All of these would have to be changed in the example repo as I will link to them as soon as it is made public.

Reply:
The yamls under examples>word-count are temporary and will be linked from the example repo when it's made public. I will update them there, noted [here](https://github.com/bakdata/kpops-examples/issues/6)